### PR TITLE
Add note to Routes documentation, that it does not support any url ro…

### PR DESCRIPTION
**Pull request by genbit**
Related to issue discussed in #7 

Term "Routes" usually is used to describe url related routing. I think adding a Note to Routes documentation, should save people time to understand that routes are about root queries and not url routing.

### CODE REVIEW

File: docs/Guides-Routes.md [hunk 1]
```diff
@@ -120,7 +122,11 @@ But now, we can also create routes for arbitrary user IDs. For example, if we wa
...
 window.addEventListener('popstate', () => {
   var userID = getQueryParamFromURI('userID', document.location.href);
-  var profileRoute = new ProfileRoute({userID});
-  // ...
+  var profileRoute = new ProfileRoute({userID: userID});
+  React.render(
+    <Relay.RootContainer
+      Component={UserProfile}
+      route={profileRoute}
+  />
```
Comment (yungsters) [inline docs/Guides-Routes.md]: Typo here, I think.

```
  />,
  container
);
```

File: docs/APIReference-Route.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ next: api-reference-relay-root-container
 
 Relay uses routes to define entry points into a Relay application.
 
+*Note* that Relay Routes don't really implement any URL routing-specific logic or work with History API. In the future we will maybe rename RelayRoute to be something more like RelayQueryRoots or RelayQueryConfig.
```
Comment (yungsters) [inline docs/APIReference-Route.md]: What do you think about using a block quote to call this out?

```
> Note
>
> Relay routes don't ...
```

**Comment by yungsters** [comment]
Thanks, this is great! Just one minor suggestion, and I think there might be a typo.

**Comment by genbit** [comment]
Yes, block quote definitely looks better, fixed typo as well. Thanks for sharing more info about future of routes.

File: docs/Guides-Routes.md [hunk 3]
```diff
@@ -120,7 +124,11 @@ But now, we can also create routes for arbitrary user IDs. For example, if we wa
...
 window.addEventListener('popstate', () => {
   var userID = getQueryParamFromURI('userID', document.location.href);
-  var profileRoute = new ProfileRoute({userID});
-  // ...
+  var profileRoute = new ProfileRoute({userID: userID});
+  React.render(
+    <Relay.RootContainer
+      Component={UserProfile}
+      route={profileRoute}
+  />, container);
```
Comment (yungsters) [inline docs/Guides-Routes.md]: Styling nit, can you indent `/>` and `);` to match the indentation of `<Relay.RootContainer` and `React.render(`, respectively? And line break all arguments (prefer all or none, I should add this to `CONTRIBUTING.md`).

**Comment by genbit** [comment]
fixed indentation

**Comment by yungsters** [comment]
Thanks again!

**Pull request merged**